### DISCUSSION TITLE
Examples - Buildless

### DISCUSSION THREAD

**Discussion by hwclass** [vercel/vercel#4005] [General] [answered]
Hi there 👋 

I am planning to add a now example showing how to deploy apps without a transpiling/build/bundling step using some bundler(s). This example will be a tiny app containing a nice combination of [preact](https://preactjs.com/) & [snowpack](https://www.snowpack.dev/) by using [covid19 public api](https://documenter.getpostman.com/view/10808728/SzS8rjbc?version=latest#00030720-fae3-4c72-8aea-ad01ba17adf8).

Is that OK to send a PR for that example?

_Note: We've already published an [app](https://buildless.site) based on the same setup, so that's all._

Thanks.

**Comment by styfle** [comment, marked as answer]
Hi @hwclass 

The official examples are reserved for frameworks and their corresponding boilerplate.

Since we already have a [preact](https://github.com/zeit/now/tree/master/examples/preact) example, we don't need another one, otherwise we would overwhelm new users selecting a template from our import page: https://zeit.co/import/templates

**Comment by clintharris** [comment]
@hwclass I would be interested in seeing this example. Do you have a repo you can share?

**Comment by clintharris** [comment]
In case anyone else finds there way here looking for a Vercel + Snowpack example: https://github.com/clintharris/playground-monorepo/tree/master/vercel-snowpack-react

**Reply by michaelwschultz** [reply]
@clintharris this was SUPER helpful! Really appreciate you leaving the link here. I was tearing my hair out!

**Reply by michaelwschultz** [reply]
Oops, I spoke too soon. Just tried this solution but I'm unable to hit the endpoint locally. :( Any idea why the Vercel dev-server would be complaining? I'm using the package.json and tsconfig as you.

localhost:3000/api/get-weather
![Screen Shot 2020-07-28 at 4 36 33 PM](https://user-images.githubusercontent.com/57668/88739871-83315c00-d0f0-11ea-8bb6-f25056f337ad.png)

Error
![Screen Shot 2020-07-28 at 4 28 45 PM](https://user-images.githubusercontent.com/57668/88739817-5c732580-d0f0-11ea-9e74-c0bd582035a3.png)

**Reply by michaelwschultz** [reply]
Going to pull down your example and see if it works for me locally in the meantime.

**Reply by michaelwschultz** [reply]
Looks like the comments in your tsconfig were blowing up for me. Fixed by removing them.

**Comment by hwclass** [comment]
Hi @clintharris,

The repo we built [buildless.site](https://buildless.site/) is private and deployed on vercel. I can give you an invite to have a look if you need. You can also find the examples which are not directly related with vercel there inside the website.

**Reply by michaelwschultz** [reply]
Have you and @clintharris tried out the new Snowpack 3 plugin system? Currently there isn't a Vercel plugin. :(
